lib/cn/cn_tree_compact.h

```diff
@@ -115,8 +115,10 @@ cn_comp_rule2str(enum cn_comp_rule rule)
     return "unknown_rule";
 }
 
-#define CW_DEBUG_ROOT 0x01 /* include ingest and root spills */
-#define CW_DEBUG_PROGRESS 0x02
+#define CW_DEBUG_PROGRESS   0x01
```

Comment:
I usually think (1 << X) is a cleaner way of creating flags.

Reply:
Either way. We should probably just be consistent about it because I have been doing ti one way and you have been doing another.

Reply:
I'll admit, it's pretty sexy, and I don't care either way until I actually have to debug the code, at which point the profanities fly when I have to convert the bit-shifted values to match the hex dump :-)